features/post/post-comment/post-comment.module.scss

```diff
@@ -0,0 +1,80 @@
+.comment {
+  display: flex;
+  gap: 5px;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  
+  &__text {
```

Comment:
In the application, lowerCamelCase is used everywhere. I would prefer it to be used everywhere.